internal/core/build_info_test.go

```diff
@@ -14,6 +14,7 @@ import (
 var fakeCommand = &Command{
 	Namespace: "plop",
 	ArgsType:  reflect.TypeOf(args.RawArgs{}),
+	NoClient:  true,
```

Comment:
Should we use a term such as `AllowAnonymousClient` ?

Reply:
tracked in #968